package.json

```diff
@@ -2167,6 +2167,7 @@
         "ts-loader": "^9.5.1",
         "typescript": "^5.4.4",
         "typescript-eslint": "^7.8.0",
+        "vscode-languageserver-types": "^3.17.5",
```

Comment:
does not seem worth this extra dependency for a single call

Reply:
Yeah that's fair. I figured it was nbd because it's just types but I'll remove it